app/scripts/analysis/GetData/GetDataMainCtrl.controller.coffee

```diff
@@ -207,7 +275,7 @@ module.exports = class GetDataMainCtrl extends BaseCtrl
       throw err
     )
 
-  getSocrData: ->
+  getSocrDataset: ->
```

Comment:
@alxndrkalinin getSocrDataset is the new method signature that is not present in `dev`, but will be after a merge